<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightTableWindow</class>
 <widget class="QWidget" name="LightTableWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>150</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Light Table</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="mainControls">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QSpinBox" name="numBeforeSpinner">
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="lightTableOnButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../blit.qrc">
           <normaloff>:/lightbulb_icon</normaloff>:/lightbulb_icon</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="lightTableLoopButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../blit.qrc">
           <normaloff>:/loop_icon</normaloff>:/loop_icon</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="numAfterSpinner">
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QSlider" name="fadeStepSlider">
         <property name="minimum">
          <number>100</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="pageStep">
          <number>50</number>
         </property>
         <property name="value">
          <number>300</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../blit.qrc"/>
 </resources>
 <connections/>
</ui>
